app/scripts/modules/core/application/listExtractor/listExtractor.service.js

```diff
@@ -8,9 +8,12 @@ module.exports = angular
   ])
   .factory('appListExtractorService', function(_) {
 
-    let getRegions = (appList) => {
+    let defaultAccountFilter = (/*cluster*/) => true;
+
+    let getRegions = (appList, accountFilter = defaultAccountFilter) => {
       return _(appList)
```

Comment:
Is `appList` the same as the `application`? If so, you should be able to skip the `clusters` and go straight to the `serverGroups`, i.e.
```javascript
let getRegions = (appList, accountFilter = defaultAccountFilter) => {
  return _(appList.serverGroups)
    .filter(accountFilter)
    .map('region')
    .unique()
    .value()
    .sort();
};
```